espnet2/utils/build_dataclass.py

```diff
@@ -12,6 +12,6 @@ def build_dataclass(dataclass, args: argparse.Namespace):
             raise ValueError(
                 f"args doesn't have {field.name}. You need to set it to ArgumentsParser"
             )
-        check_type(field.name, getattr(args, field.name), field.type)
+        check_type(getattr(args, field.name), field.type)
```

Comment:
Is this OK?
(I just want to confirm)

Reply:
The `checktype` from the last version of `typeguard` only accepts two inputs: the variable and its type, the name is no longer required.
Ref: <https://typeguard.readthedocs.io/en/stable/userguide.html>